src/schema/objects/columns.yaml

```
@@ -123,6 +137,7 @@ handedness:
   - n/a
 hemisphere:
   name: hemisphere
+  display_name: Electrode hemisphere
```

AFAIK electrodes have no hemispheres... 

```suggestion
  display_name: Electrode placement hemisphere
```